packages/nodejs/.changesets/support-opentelemetry-root-spans-in-spanprocessor.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "change"
+---
+
+Support OpenTelemetry root spans in SpanProcessor. This change makes AppSignal instrumentation (like Express/Koa.js/Next.js) no longer a requirement. In fact you will need to use the OpenTelemetry instrumentation for those libraries from now on.
```

Comment:
It seems to imply that you **must** switch to OT-based Express/Koa.js/Next.js instrumentation now, period. You only need to do that in order to use other OpenTelemetry instrumentations.

```suggestion
Support OpenTelemetry root spans in SpanProcessor. This change makes AppSignal instrumentation (like Express/Koa.js/Next.js) no longer a requirement. In order to use OpenTelemetry-based integrations such as MySQL, you will need to use the OpenTelemetry instrumentation for these libraries from now on.
```

> In fact you will need to use the OpenTelemetry instrumentation for those libraries from now on.

... Isn't this kind of a big problem? This means that our integrations are no longer compatible with each other. We now have two flavours of integrations who can't meaningfully inter-operate with each other. If you want to instrument Express and IORedis, you can't also instrument Postgres.